docs/nontech-explainer.md

```diff
@@ -0,0 +1,65 @@
+# Openness in Tech: An Explainer With NO Tech Jargon
+
+> What openness means, and why we care
+
+## Why We Need Openness
+
+In its broader sense, openness is mostly defined by negation. "Being open" means to "not be" closed, as boundaries allow to define closeness more clearly.
+
+The same idea goes for the more specific sense of openness we find in IT. IT mostly took shape during the cold war with the emergence of a "closed-world discourse[^1]" emphasizing security through obscurity and global surveillance. This discourse strongly influenced our vision of IT, and continues to do so today. We still tend to see code, innovation, and information as something we need to protect. We still think we need some kind of magical secret on which we could build benefits.
+
+But the era of secrets is long gone. Now, people expect access to all the information they need, whenever they need. Secrecy in business and technology is considered suspicious. Transparancy is key.
+
+And it goes both ways. Customers are now using social platforms on a daily basis, constantly sharing and collaborating with their peers. Here again, secrecy is a far memory, but without necessarily leading privacy to its end. The public expect a control proportionnal to the amount of data it shares. It can't accept to be passive anymore. Companies now have to offer everyone both control over their data, and opportunities to actively contribute.
```

Comment:
> The public expect a control proportionnal to the amount of data it shares.
I find this phrase complicated. The idea of the **public** beign the subject and then the "it shares" makes this complicated - the subject is an **it** who is very active, expects/accepts...

Maybe "the modern user expects..."

Reply:
I've split this sentence for more clarity, but not sure if the second part is grammatically correct and clear enough.
Please tell me what you think.